scripts/Package_BGA/bga_xilinix.yml

```diff
@@ -0,0 +1,148 @@
+# Artix-7 FPGA Packages
+# Notes:
+#  - Often names are similiar, e.g. FB, FBG and FBV. I choose FBG as it is the common one.
+#    The addition stems from different solder types for the BGA balls
+#  - Xilinx_CPG236 and Xilinx_CPG236 are the same, different name comes from bank differences
```

Comment:
The two packages listed here are the same but should be different.

Reply:
You are absolutely correct.
The second name should be Xilinx_CPG238.

The change is already pushed.
Thanks for the fast feedback!